genie-ddl/src/main/sql/mysql/schema.mysql.sql

```diff
@@ -1,8 +1,8 @@
--- MySQL dump 10.13  Distrib 5.7.17, for osx10.12 (x86_64)
+-- MySQL dump 10.13  Distrib 5.7.17, for macos10.12 (x86_64)
 --
 -- Host: localhost    Database: genie
 -- ------------------------------------------------------
--- Server version	5.7.17
+-- Server version	5.6.35
```

Comment:
Yes, I used MAMP because it's a breeze to install, but it's not the newest.

However since it created a dump pretty much identical to the previous one, and the new table is identical in schema to application_dependencies, makes me think this is ok